main.py

```diff
@@ -18,4 +18,15 @@ def read_item(item_id: int, q: Union[str, None] = None):
 @app.post("/orders", response_class=PlainTextResponse)
 def get_order(order_details: order.Order):
     order.save_order(order_details)
-    return f"order saved\n"
\ No newline at end of file
+    return f"order saved\n"
+
+@app.get("/price")
+def calculate_price(miles: int,
+                    pallets: int = 0,
+                    packages: int = 0):
+    markup = 0.5
+    pallet_cost_per_mile = 0.0683053788345865
```

Comment:
Please use constants for all variables like this one. This way you can reuse them in different methods. To do so, put it in the beginning of the file under imports in upper case.

```
PALLET_COST_PER_MILE = 0.0683053788345865
```

Reply:
Agree with @Zhenia-Magic here, no magic numbers